packages/config/config/devices/0x0086/zw141.json

```diff
@@ -0,0 +1,229 @@
+// Aeotec ZW141
+// Nano Shutter
+{
+	"manufacturer": "Aeotec",
+	"manufacturerId": "0x0086",
+	"label": "ZW141",
+	"description": "Nano Shutter",
+	"devices": [
+		{
+			"productType": "0x0303",
+			"productId": "0x008d",
+			"zwaveAllianceId": 3132
+		},
+		{
+			"productType": "0x0903",
+			"productId": "0x008d",
+			"zwaveAllianceId": 3269
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"35": {
+			"label": "Moving Time From Up (left) to Down (right) For Curtain",
+			"description": "The available range for time set is 1 to 127 seconds",
```

Comment:
We don't need the available range in the description as we can see the min/max values and the unit. Please remove this one and any others.